.gitignore

```diff
@@ -32,6 +32,13 @@ phpcs.xml
 /docker/wordpress-develop/*
 !/docker/wordpress-develop/.gitkeep
 /docker/wordpress/*
+!/docker/wordpress/wp-content
+/docker/wordpress/wp-content/*
+!/docker/wordpress/wp-content/plugins
+/docker/wordpress/wp-content/plugins/*
+!/docker/wordpress/wp-content/plugins/a
```

Comment:
I guess we're not planning to land this?

Reply:
we are not, it is just a way of sharing a plugin to test with